build.assets/Makefile

```diff
@@ -121,7 +121,7 @@ build-binaries-fips: buildbox-fips
 buildbox:
 	if [[ "$(BUILDBOX_NAME)" == "$(BUILDBOX)" ]]; then \
 		if [[ $${DRONE} == "true" ]] && ! docker inspect --type=image $(BUILDBOX) 2>&1 >/dev/null; then docker pull $(BUILDBOX) || true; fi; \
-		docker build --platform=linux/$(RUNTIME_ARCH) \
+		 DOCKER_BUILDKIT=1 docker build --platform=linux/$(RUNTIME_ARCH) \
```

Comment:
There’s also a leading space here; doesn’t make any difference to the way it runs, just can’t unsee it 😁